README.md

```diff
@@ -489,16 +489,17 @@ Nexus instance that hosts the permanent content: `cachito_nexus_hoster_username`
 
 The table below shows the supported package managers and their support level in Cachito.
 
-Feature                 | gomod | npm | pip |
----                     | ---   | --- | --- |
-Baseline                | ✓     | ✓   | ✓   |
-Content Manifest        | ✓     | ✓   | ✓   |
-Dependency Replacements | ✓     | x   | x   |
-Dev Dependencies        | ✓     | ✓   | ✓   |
-External Dependencies   | N/A   | ✓   | ✓   |
-Multiple Paths          | x     | ✓   | ✓   |
-Nested Dependencies     | ✓     | ✓   | x   |
-Offline Installations   | ✓     | x   | x   |
+Feature                 | gomod | npm | pip | git-submodule |
```

Comment:
Not sure if we should include `git-submodule` here, it's not really a full-featured package manager, more like a secondary thing that would usually be used in combination with other package managers.

IMO we could just include the relevant info in the `### git-submodule` section below

Reply:
I thought a lot about this as well. The only reason I ended up choosing this approach was to make sure, each possible value for `pkg_managers` is part of the table as in to give an overview, hence I also put a `+` there to explain the specifics of it but the other part of me agrees with you. So, I'll just remove it